src/components/PageSwitch.tsx

```diff
@@ -21,7 +21,8 @@ import { useCallback, useEffect, useMemo } from 'react'
 import { useLocation, useNavigate } from 'react-router-dom'
 import styled from 'styled-components'
 
-import useQueryParams from '../hooks/useQueryParams'
+import useQueryParams from '@/hooks/useQueryParams'
+
 import TextButton from './Buttons/TextButton'
```

Comment:
I guess it's not important that some imports have not been updated, right? We can do that step by step in future PRs as we notice old-fashioned imports 😉

Reply:
Exactly